README.md

```diff
@@ -16,11 +16,11 @@ A Jenkins Plugin that supports authentication & authorization via Azure Active D
 
 1. Click `Authentication`, under 'Implicit grant and hybrid flows', enable `ID tokens`.
 
-1. (optional) To enable AzureAD group support: Click `Manifest` and modify the `"groupMembershipClaims": null` value to `"groupMembershipClaims": "SecurityGroup"`, then 'Save' it.
+1. (optional) To enable Entra ID group support: Click `Manifest` and modify the `"groupMembershipClaims": null` value to `"groupMembershipClaims": "SecurityGroup"`, then 'Save' it.
```

Comment:
> It seems Microsoft always calls it "Microsoft Entra ID" rather than just "Entra ID".

Most of the times yes, but there are exceptions when the context is clear:
https://learn.microsoft.com/en-us/entra/fundamentals/whats-new#:~:text=provisioning%20into%20Entra%20ID

> With API-driven inbound provisioning, Microsoft Entra ID provisioning service now supports integration with any system of record. Customers and partners can use any automation tool of their choice to retrieve workforce data from any system of record for provisioning into **Entra ID** and connected on-premises Active Directory domains. The IT admin has full control on how the data is processed and transformed with attribute mappings. Once the workforce data is available in **Entra ID**, the IT admin can configure appropriate joiner-mover-leaver business processes using **Entra ID** Governance Lifecycle Workflows.

Anyway, I'll update to use the full name.